plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -3159,6 +3159,24 @@ else if (object.getBukkitEntity() instanceof Hanging hanging) {
             }
         });
 
+        // <--[mechanism]
+        // @object EntityTag
+        // @name is_sleeping
+        // @input EntityTag
+        // @description
+        // Sets whether a fox is sleeping.
+        // The entity may wake up immediately after setting this to true. If this is not desired, disable <@link mechanism has_ai>.

```

Comment:
> `<@link mechanism has_ai>`

Need to specify the full mechanism there, `EntityTag.has_ai`.
Also this should mayprobably be a property link?